src/main/java/Duke.java

```diff
@@ -1,10 +1,95 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+import java.io.IOException;
+import java.util.*;
+import java.io.File;
+
+
+
+public class Duke{
+    public static void main(String[] args) throws IOException,Exception {
+        Scanner sc = new Scanner(System.in);
+        String mesInput = "";
+        ArrayList<Task> arrTask = new ArrayList<Task>();
+//        TaskList tList = new TaskList(arrTask);
+        Ui uiDisplay = new Ui();
+        Storage  fileStorage = new Storage();
+        String filePath = "duke.txt";
+        File f = new File(filePath);
+
+        while(!mesInput.equalsIgnoreCase("bye")){
```

Comment:
Might be simpler to read the input once before the loop, and then again at the end of the loop, to minimise the number of "bye"-checking statements.